prez/sparql/objects_listings.py

```diff
@@ -79,11 +79,17 @@ def generate_listing_construct(
     # based on them being an instance of a class), else use the URI of the "parent" off of which members will be listed.
     # TODO collapse this to an inline expression below; include change in both object and listing queries
```

Comment:
can this code be moved to the generate_sequence_construct function so as to make the main query construction method shorter